invision-links.md

```diff
@@ -63,7 +63,7 @@ title: Fabric8 UXD
 | Space Settings - Research and Wireframes          | Wireframe             | Unassigned    | Not Yet Started   |                   |                                                                                                          | [UX GitHub Issue 232](https://github.com/fabric8io/fabric8-ux/issues/232)
 | Tags - Desktop                                    | Wireframe             | Sj            | Ready for Dev     | 10/27/2016        | [https://redhat.invisionapp.com/share/6E8ZFBF2Q](https://redhat.invisionapp.com/share/6E8ZFBF2Q)         |
 | Tags - Desktop                                    | Visual                | Unassigned    | TBD               |                   |                                                                                                          | [UX GitHub Issue 244](https://github.com/fabric8io/fabric8-ux/issues/244)
-| User Profile                                      | Wireframe             | Unassigned    | Not Yet Started   |                   |                                                                                                          | [UX GitHub Issue 301](https://github.com/fabric8io/fabric8-ux/issues/301)
+| User Profile                                      | Wireframe             | Jessica       | Ready for Dev     | 04/05/2017        | [https://redhat.invisionapp.com/share/KQB6LEHM8](https://redhat.invisionapp.com/share/KQB6LEHM8)         | [UX GitHub Issue 301](https://github.com/fabric8io/fabric8-ux/issues/301)                                                                                                        | [UX GitHub Issue 301](https://github.com/fabric8io/fabric8-ux/issues/301)
```

Comment:
@jryhanyc can you remove one of the `[UX GitHub Issue 301](https://github.com/fabric8io/fabric8-ux/issues/301)` lines? it adds another column